### PR TITLE
Fix error with itk::LabelOverlapMeasuresImageFilter::SetInput

### PR DESCRIPTION
**Pull request by blowekamp**
The ITK refactoring of this class included a change of base class,
removing the SetInput method used in SimpleITK. The JSON is updated to
use named inputs.

### CODE REVIEW

**Comment by blowekamp** [comment]
@jamesobutler This shoulda dress the  compilation error encountered when building Slicer. The more interesting and harder questions is how implicit cast was working on other builds and when the SimpleITK CI system didn't reproduce this error.

**Review by jamesobutler — COMMENTED** [review]
These changes are to be compatible as of the ITK changes made in https://github.com/InsightSoftwareConsortium/ITK/commit/83480a0aa35dbaca44354d945a4e3601059e894c?

I'm currently building to test this fix.

**Comment by blowekamp** [comment]
Yes, I testing against v5.3rc04. It may work with the old code too, but that was not tested.

File: Code/BasicFilters/json/LabelOverlapMeasuresImageFilter.json [hunk 1]
```diff
@@ -2,11 +2,22 @@
...
   "template_test_filename" : "ImageFilter",
-  "number_of_inputs" : 2,
+  "number_of_inputs" : 0,
   "pixel_types" : "IntegerPixelIDTypeList",
   "filter_type" : "itk::LabelOverlapMeasuresImageFilter<InputImageType>",
   "no_procedure" : true,
   "no_return_image" : true,
+  "inputs" : [
+    {
+      "name" : "SorceImage",
```
[the remark this answers is not in the document]
Comment (jamesobutler) [reply Code/BasicFilters/json/LabelOverlapMeasuresImageFilter.json:12]: @blowekamp This need to be fixed up for the testing to be successful?

**Comment by jamesobutler** [comment]
I had no build errors with the changes in this PR 👍🏻. Prior to this PR I was facing the error described at https://github.com/Slicer/Slicer/pull/6454#issuecomment-1230151624.

@blowekamp You going to create a v2.2.1 tag following up soon? I notice the v2.2.0 tag, but not an actual release generated for it yet.

**Comment by blowekamp** [comment]
SimpleITK 2.2.0 is going to be announce today.

@jamesobutler The current plan is to create the v2.2.1 tag when ITK v5.3.0 is tagged and release. There should be some time before the next patch/tweak increment to allow for other issues to be reported and addressed. Hopefully, the ITK tag will occur with in a month, if its further delayed then anther patch release may be created before.

**Review by zivy — APPROVED** [review]

**Pull request merged**